docs/rhel4edge_iso.md

```diff
@@ -56,9 +56,11 @@ Optional arguments:
           included in the image (default: none)
   -embed_containers
           Embed the MicroShift container dependencies in the image
-  -ostree_server_name name_or_ip
-          Name or IP address and optionally port of the ostree
-          server (default: 127.0.0.1:8080)
+  -ostree_server_url URL
+          URL of the ostree server (default: http://127.0.0.1:8080/repo)
```

Comment:
We might want to pick a different port by default to leave that common port for users' apps. That would also let us pick a port that is likely to have firewall rules blocking access from outside the host, too.

Reply:
Good point. Updated to 8085, which does not seem to be allocated to any "known" service.